dev/env.go

```diff
@@ -79,6 +67,14 @@ func NewEnvironment(name, workDir string, opts ...EnvironmentOption) *Environmen
 
 // Initializes the environment and prepares it for use.
 func (env *Environment) Init(ctx context.Context) error {
+	if env.config.ContainerRuntime == ContainerRuntimeAuto {
```

Comment:
Do you propose the getter function on the EnvironmentConfig struct or on the Environment struct?
What do you want to accomplish with this change?

Reply:
setContainerRuntime make sense instead of get

func (env *Environment) Init(ctx context.Context) error {

   setContainerRuntime(env) 
 --

}

func setContainerRuntime(env *Environment) error {
    if env.config.ContainerRuntime == ContainerRuntimeAuto {
                 cr, err := DetectContainerRuntime()
		if err != nil {
			return err
		}
                env.config.ContainerRuntime = cr
   }
   return nil
}